src/test/java/org/broadinstitute/hellbender/testutils/VariantContextTestUtilsUnitTest.java

```diff
@@ -88,10 +88,10 @@ public void testNormalizeScientificNotation(Object toNormalize, Object expected)
     public Object[][] getIntegerValuesToNormalize(){
         final Object aSpecificObject = new Object();
         return new Object[][] {
-                {"27", new Integer(27)},
-                {"-27", new Integer(-27)},
-                {"0", new Integer(0)},
-                {"-27014", new Integer(-27014)},
+                {"27", Integer.valueOf(27)},
```

Comment:
I was thinking this might have been a deliberate choice, but looking at it closely I don't think so.  This seems fine.